### PR TITLE
Add color

### PR DESCRIPTION
**Pull request by Rahaf91**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
+import ColorForm from "./Components/ColorForm/ColorForm.jsx";
+import { useState } from "react";
+import { uid } from "uid";
 
 function App() {
+  const [colors, setColors] = useState(initialColors);
+
+  function handleAddColor(newColor) {
+    setColors([{ id: uid(), ...newColor }, ...colors]);
+    console.log("Adding new color:", newColor);
```
Comment (JessicaLoers) [inline src/App.jsx:13]: Please remove the `console.log` statement before merging to keep the production code clean and avoid unnecessary logging. 🧹

```suggestion
```

File: src/Components/ColorForm/ColorForm.css
Comment (JessicaLoers) [inline src/Components/ColorForm/ColorForm.css:1]: Please remove empty files before merging to keep the production code clean.

File: src/Components/ColorInput/ColorInput.css
Comment (JessicaLoers) [inline src/Components/ColorInput/ColorInput.css:1]: Please remove empty files before merging to keep the production code clean.

**Review by JessicaLoers — APPROVED** [review]
**Hi @Rahaf91** 

Just a few minor suggestions: please delete all empty files and remove the console.log statement. 

**Keep up the good work!**

File: src/App.jsx (hunk 1 above)
Comment (Rahaf91) [reply src/App.jsx]: Hi @JessicaLoers 
Thank you for the review . I have just checked the output in the console and forgot to delete it. I deleted **colorInput. css** file and kept the **colorForm.css** to which I added some styling.

**Pull request merged**